pipenv/cli.py

```diff
@@ -306,8 +306,8 @@ def ensure_pipfile(validate=True):
     if project.pipfile_is_empty:
 
         # If there's a requirements file, but no Pipfile...
-        if project.requirements_exists:
-            click.echo(crayons.normal(u'requirements.txt found, instead of Pipfile! Converting…', bold=True))
+        if project.requirements_exists and not skip_requirements:
+            click.echo(crayons.normal(u'Requirements.txt found, instead of Pipfile! Converting…', bold=True))
```

Comment:
Only issue I have here is this line. `requirements.txt` is the files name and shouldn't really be capitalized by convention.